vrecord

```diff
@@ -462,23 +462,9 @@ lookup_standard(){
     esac
 }
 
-lookup_framemd5(){
+yes_or_no(){
     case "${1}" in
-        "Yes")
-            ;;
-        "No")
-            ;;
-        "quit") _report -d "Bye then" ; exit 0 ;;
-        *) _report -w "Error: Not a valid option, select a valid number." ; return 1 ;;
-    esac
-}
-
-lookup_qctoolsxml(){
-    case "${1}" in
-        "Yes")
-            ;;
-        "No")
-            ;;
+        "Yes")|"No") ;;
```

Comment:
The `"Yes")|"No")` syntax doesn't seem to work (I get the response: ``line 467: syntax error near unexpected token `|'`` from Terminal), but if I change it to `"Yes"|"No")` it stops complaining.

Reply:
Yes, of course, my typo… Thanks a lot!